CHANGELOG-0.11.md

```diff
@@ -0,0 +1,18 @@
+# Changelog 0.11
+
+## [0.11.0] 2021-xx-xx
+
+### Added
+
+- [#2033](https://github.com/epiphany-platform/epiphany/issues/2033) - Add arm64 support to 'repository' component
+- [#2109](https://github.com/epiphany-platform/epiphany/issues/2109) - Add arm64 support to 'kubernetes_master' and 'kubernetes_node' components
+- [#2110](https://github.com/epiphany-platform/epiphany/issues/2111) - Add arm64 support to 'kafka' component
+- [#2111](https://github.com/epiphany-platform/epiphany/issues/2111) - Add arm64 support to 'load_balancer' component
```

Comment:
```suggestion
#### ARM

- [#2033](https://github.com/epiphany-platform/epiphany/issues/2033) - Add arm64 support to 'repository' component
- [#2109](https://github.com/epiphany-platform/epiphany/issues/2109) - Add arm64 support to 'kubernetes_master' and 'kubernetes_node' components
- [#2110](https://github.com/epiphany-platform/epiphany/issues/2111) - Add arm64 support to 'kafka' component
- [#2111](https://github.com/epiphany-platform/epiphany/issues/2111) - Add arm64 support to 'load_balancer' component
```

Reply:
How about we mark them with [ARM64] in the title as we do with OS?

Reply:
Sounds good as well.

Reply:
Fixed